.drone.star

```diff
@@ -603,25 +605,15 @@ def setGuiTestReportDir():
         ],
     }]
 
-def showGuiTestResult():
-    return [{
-        "name": "show-gui-test-result",
-        "image": PYTHON,
-        "commands": [
-            "python %s/TestLogParser.py %s/results.json" % (dir["guiTest"], dir["guiTestReport"]),
-        ],
-        "when": {
-            "status": [
-                "failure",
-            ],
-        },
-    }]
-
 def uploadGuiTestLogs(ctx, server_type = "oc10"):
     trigger = {
         "status": [
             "failure",
         ],
+        "event": [
+            "cron",
+            "tag",
+        ],
```

Comment:
we may want reports on PR failures to know exactly what failed. So instead of gh-comment, adding ci step to log reports url might be useful.